src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+@use '../_variables/variables' as *;
+@use '../mixins/mixins.scss' as *;
+
+.card {
+  width: $CardWidth;
```

Comment:
The variable `$CardWidth` should be written in lowercase letters with hyphens instead of camelCase, as per the code style rules.